### ISSUE TITLE
IDEAS

### ISSUE THREAD

**Issue by garcharjay811** [oss2019/celebrino#13]
Need new Ideas and suggestions regarding the website, like something which could be added to make it better or any suggestions.

**Comment by bvishal8510** [comment]
@garcharjay811 I was going through the website and found some issues that can be improved.
1. There is the latest event section on the home page and there is a separate event page. What can be done is. The event button in the header can lead to the latest event section where we can have an anchor tag(more events) which leads to the events page.
2. On Team page, along with photo and name, some details of members can be shown. It will look good.

**Comment by samagragupta** [comment]
@bvishal8510 You can create an issue and work on them or suggest a change on other issues

**Comment by garcharjay811** [comment]
@bvishal8510  Yes, as you said ( in idea 1 ) essentially the idea is the same, we will show some featured events in the home page and all the events in a separate page. As @samagragupta said you can open an issue and can start working on it. BTW it's great that you can up with ideas.
